packages/compass-generative-ai/src/atlas-ai-service.ts

```diff
@@ -12,6 +12,7 @@ import { EJSON } from 'bson';
 import { signIntoAtlasWithModalPrompt } from './store/atlas-signin-reducer';
 import { getStore } from './store/atlas-ai-store';
 import { optIntoGenAIWithModalPrompt } from './store/atlas-optin-reducer';
+import { throwIfNotOk } from '../../atlas-service/src/util';
```

Comment:
When importing something from another package we follow the npm package import syntax. So we should update how we're exporting this from atlas-service to make this import like:
```suggestion
import { throwIfNotOk } from '@mongodb-js/atlas-service/renderer';
```